fixtures/ignition/2bright-par-18-ip.json

```diff
@@ -0,0 +1,416 @@
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [26, 28],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [29, 31],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [32, 34],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [35, 37],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [38, 40],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [41, 43],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [44, 46],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [47, 49],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [50, 52],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [53, 55],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [56, 58],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [59, 61],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [62, 64],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [65, 67],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [68, 70],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [71, 73],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [74, 76],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [77, 79],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [80, 82],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [83, 85],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [86, 88],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [89, 91],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [92, 94],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [95, 97],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [98, 100],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [101, 103],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [104, 106],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [107, 109],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [110, 112],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [113, 115],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [116, 118],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [119, 121],
+          "type": "ColorPreset"
+        },
+        {
+          "dmxRange": [122, 125],
+          "type": "ColorPreset"
+        },
```

Comment:
These DMX values don't match the manual:

![image](https://user-images.githubusercontent.com/202916/173042714-041c9cc8-75b8-4575-85c6-dc646a767727.png)

Also, you added 38 `ColorPreset` capabilities, while the manual only mentions 35.

@hrueger Could you please check the exact DMX values for all color presets? An extra bonus would be to know the specific colors :wink:

Reply:
Hi, sure. I just haven't opened the parcel yet ;-) but I'll fix that as soon as possible and see what colors I get.
Thanks for fixing the other things, I didn't really know how to specify the strobe channel's capabilities correctly.

Reply:
I'll do that on Monday.

Reply:
So this is what I observed...
´´´
Some more things I noticed when testing the fixture:
- in 2ch mode, the fixture is off (even when the dimmer channel is at 100%) if the color temperature channel is 0 - 5. 


color macros:
0 white
6 off
11 red
14 dark orange
17 yellow
20 green
23 light blue
26 dark blue
29 pink
32 brighter pink
35 green / yellow mix
38 grey (dimmed white)
41 red / orangeish
44 red / more orangeish
46 red / even more orangeish
50 red / even even more orangeish
53 orange
59 dimmed orange
62 dimmed yellow
65 greenish yellow
68 more greenish yellow
71 green / slightly turquoise
74 green / more turquoise
77 turquoise
80 light blue
83 sky blue
86 dark blue
89 pale pink
92 more pink
95 bright pink
98 dark red
101 grey
106 lighter grey
111 white / slightly blueish
116 white / more blueish
121 UV

rest is in the manual
```

I wasn't really able to find suiting hex colors as they look so different on the screen than light shining on a wall.

Are you fine with this information?

Reply:
Thanks, I added the colors in [`001b9fd`](https://github.com/OpenLightingProject/open-fixture-library/pull/2662/commits/001b9fdbd532c2efc76e7fdb7fa134057097ad5e).

Don't worry about colors not looking the same as the fixture; that is dependent on so many factors like environment lighting, display quality, fixture CRI, …
The RGB values in the fixture definitions are meant as a help for fixture operators / light show programmers. A (rough) color representation helps to choose a color capability much quicker than having to read and parse the capability name.